mc_all.sas

```diff
@@ -1989,10 +1989,10 @@ data datalines_2;
     ('PROCESSED_DTTM','VALID_FROM_DTTM','VALID_TO_DTTM')));
   if type='num' then dataline=
     cats('ifc(int(',name,')=',name,'
-      ,put(',name,',best32.)
-      ,substrn(put(',name,',bestd32.),1
-      ,findc(put(',name,',bestd32.),"0","TBK")))');
-  else dataline=cats('strip(',name,')') ;
+      ,put(',name,',best32.-l)
+      ,substrn(put(',name,',bestd32.-l),1
+      ,findc(put(',name,',bestd32.-l),"0","TBK")))');
```

Comment:
Nice, hadn't seen the `-l` option before!  Does it work on 9.3?

Reply:
yes - back to 9.2.  Will left align:  https://support.sas.com/documentation/cdl/en/lrdict/64316/HTML/default/viewer.htm#a000199354.htm

Reply:
yes, left align values, so later we use `t` for trim in CATQ function instead of `s`